### PR TITLE
APIGOV-23279 - Paid plan documentation

### PR DESCRIPTION
**Pull request by cbordier**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Add the paid plan information

## Checklist for contributors

Before submitting this PR, please make sure:

* [X] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [X] You have signed the [Axway CLA](https://cla.axway.com/)
* [X] You have verified the technical accuracy of your change
* [X] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [X] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

**Comment by lbadenhop** [comment]
@cbordier I'll review as soon as Georgi approves.

File: content/en/docs/manage_product_foundry/manage_product_plans.md [hunk 1]
```diff
@@ -20,18 +20,61 @@ Learn how to create and configure the product plan using the Product Foundry Web
...
-## Create a free product plan
+## Product plan type and consumption cost
 
-When you create a new product, you have the option to configure a **Free** product plan. If you skip this step, a default free plan will be auto generated, which you can edit / modify later.
+There are 2 types of plans that can be used:
 
-To configure a free product plan:
+* **free plan**: consumer will not be charged for their API consumption
+* **paid plan**:
+    * **standard**: consumer will be charged based on the volume of their consumption
```
Comment (georgiana-caras) [inline content/en/docs/manage_product_foundry/manage_product_plans.md:36]: **standard**: consumer will be charged a base price based for the consumption of a guaranteed set of quota.

File: content/en/docs/manage_product_foundry/manage_product_plans.md [hunk 2]
```diff
@@ -40,14 +83,24 @@ To configure a free product plan:
...
+To configure a product plan from the product detail:
+
+1. Navigate to the Plans section
+2. Click the **+ Add Plan** button to open the plan creation wizard
+3. Follow the plan wizard and enter the required information
 
-## Configure a quota
+### Configure a quota
 
 A quota describes the itemized units per resource or group of resources in a product, and how much of those units they are entitled to use over a period.
```
Comment (georgiana-caras) [inline content/en/docs/manage_product_foundry/manage_product_plans.md:96]: period of time.

File: content/en/docs/manage_product_foundry/manage_product_plans.md [hunk 3]
```diff
@@ -58,7 +111,34 @@ To configure a quota, enter values for the following properties:
...
+
+* **Base price**:
+    * **Currency** - the currency that will be used by the billing system (USD - EUR)
+    * **Amount** - the price value
+    * **Metering period** - select either **Monthly**, **Daily**, **Weekly**, **Annually**
+* **Quota type** - select either **Standard** or **Tiered**
+    * Standard quota type has the same information as for the free plan (Unit, Limit, Quota type, Limit type)
+    * Tiered quota type: for each tier, you need to enter the **lower limit**, the **upper limit** the **unit price** and the **Standard** fees. There is no limit in the tier number. The **+** button enables to add another tier and the **-** button enables to remove a tier definition. The lower limit a the next tier is automatically computed based the the upper limit of previous tier.
+
+The **+ Add Quota** button allows to create another quota group for different resources. Once a resources is assigned to a quota group, it is no more avialble for another quota group.
```
Comment (georgiana-caras) [inline content/en/docs/manage_product_foundry/manage_product_plans.md:124]: no longer available for use in another quota group.

File: content/en/docs/manage_product_foundry/manage_product_plans.md [hunk 4]
```diff
@@ -58,7 +111,34 @@ To configure a quota, enter values for the following properties:
...
+
+When you are finished configuring the quotas, click **Next** to select the approval mode of the plan.
+
+### Configure access
+
+To configure the type of approval when a subscriber select the plan, choose between:
```
Comment (georgiana-caras) [inline content/en/docs/manage_product_foundry/manage_product_plans.md:130]: To configure the type of approval when a subscribers submits a request to purchase the plan, choose between:

**Review by lbadenhop — APPROVED** [review]

**Pull request merged**